packages/core-logger-winston/__tests__/logger.test.ts

```diff
@@ -78,10 +96,11 @@ describe("Logger", () => {
     describe("stopTracker", () => {
         it("should stop the tracker", () => {
             logger.stopTracker("test_title", 50, 100);
+            logger.stopTracker("second_tracker", 101, 100);
 
             expect(message).toMatch(/test_title/);
             expect(message).toMatch(/=========================/);
-            expect(message).toMatch(/50/);
+            expect(message).toMatch(/100/);
```

Comment:
There should be a separate test that asserts it doesn't go over 100% instead of altering this test.